Cargo.toml

```diff
@@ -78,7 +79,7 @@ tremor-value = {path = "tremor-value"}
 url = "2.2"
 value-trait = "0.2"
 rustls = "0.19"
-async-tls = "0.11.0"
+async-tls = { version = "0.11", features = ["client", "server"]}
```

Comment:
Do we need to specify the features as these should be defaulted? Its worth checking

Reply:
```suggestion
async-tls = "0.11"
```